dev/io.openliberty.wsoc.ssl.internal/bnd.bnd

```diff
@@ -0,0 +1,26 @@
+#*******************************************************************************
+# Copyright (c) 2022 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+bSymbolicName=io.openliberty.wsoc.ssl.internal
+
+Bundle-Name: WebSocket
+Bundle-SymbolicName: io.openliberty.wsoc.ssl.internal
+Bundle-Description: WebSocket 2.1 - SSL Channel Link , version ${bVersion}
+
+javac.source: 11
+javac.target: 11
```

Comment:
This can't be java 11 if we're using it within features previous to EE10.

Reply:
Removed the javac 11 lines